mongo/manifest.json

```diff
@@ -4,7 +4,8 @@
   ],
   "categories": [
     "data store",
-    "log collection"
+    "log collection",
+    "Autodiscovery"
```

Comment:
This will create a new category on the doc site, is that expected? If so I'd say it would be nice to have all compatible integrations to use this new category, otherwise people looking for integrations with autodiscovery on the doc site might be incorrectly disappointed.

What do you think?

Reply:
Yup that's the plan, all integrations that have AD instructions are going to be flagged with autodiscovery category :)